reddit_service_websockets/dispatcher.py

```diff
@@ -22,6 +46,14 @@ def __init__(self, metrics):
     def on_message_received(self, namespace, message):
         consumers = self.consumers.get(namespace, [])
 
+        # Compress the message
+        compressed = make_compressed_frame(message, COMPRESSOR)
+        message = Message(
+            compressed=compressed,
+            raw=message,
+            percent_compressed=1 - float(len(compressed))/float(len(message)),
```

Comment:
Good point. That actually raises something more serious: sending a metric for each connection is gonna cause a big ol' storm of metrics isn't it?

Could we perhaps aggregate up the savings and send one metric instead?